evap/results/exporters.py

```diff
@@ -199,7 +219,11 @@ def write_headings_and_evaluation_info(
         # One more cell is needed for the question column
         self.write_empty_row_with_styles(["default"] + ["border_left_right"] * len(evaluations_with_results))
 
-    def write_overall_results(self, evaluations_with_results, course_results_exist):
+    def write_overall_results(
+        self,
+        evaluations_with_results: list[tuple[Evaluation, OrderedDict[int, list[QuestionResult]]]],
```

Comment:
Strong agree, the current version is not typed correctly. I would also prefer `annotated_evaluation` over `evaluation_as_any` as a naming scheme everywhere, because the former is what happens in the code, while the latter only happens in the typing and is only incidental complexity

Whether or not we end up with `Protocol` in the end, having `AnnotatedEvaluation` already typed out seems like a good idea

Reply:
annotated as any

Reply:
Yes, I saw that. What I meant was my suggestion to rename `evaluation_as_any` to `annotated_evaluation` - or do you have arguments against that?

The line `evaluation_as_any: Any = evaluation` also still does not use `AnnotatedEvaluation`

Reply:
Changed that.